config.go

```diff
@@ -142,3 +172,38 @@ func (c Config) requireTLS() bool {
 func (c Config) requireAuth() bool {
 	return c.AuthPolicyPath != "" && c.OidcIssuerURL != "" && c.OidcIssuerClientID != ""
 }
+
+func getDeleteDeadPacksTimeEnvVarWithDefault(name, defaultVal string) string {
+	val, isSet := lookupEnv(name)
+	if !isSet {
+		logger.Infof(fmt.Sprintf("%s env not set, using default %v", name, defaultVal))
+		return defaultVal
+	}
+	if err := validateTime(val); err != nil {
+		logger.Errorf(fmt.Sprintf("%s env is invalid, using default %v, error: %v", name, defaultVal, err))
+		return defaultVal
+	}
+
+	logger.Infof("Using %s=%s", name, val)
+	return val
+}
+
+func validateTime(t string) error {
+	var err error
```

Comment:
I think we can replace all this code with 
```
_, err := time.Parse("15:04", t)
return err
```

I will even remove this method an use that line above.

Reply:
Cheers, I wasn't aware of this - done